MIGRATION_GUIDE.md

```diff
@@ -1,9 +1,170 @@
 # Migration guide
 
-This document is meant to help you migrate your Terraform config to new newest version. In migration guides we will only 
-describe deprecations or breaking changes and help you to change your configuration to keep the same (or similar) behaviour
+This document is meant to help you migrate your Terraform config to the new newest version. In migration guides, we will only 
+describe deprecations or breaking changes and help you to change your configuration to keep the same (or similar) behavior
 across different versions.
 
+## vX.XX.X -> v0.85.0
+
+### Migration from old grant resources to new ones
+
+In recent changes, we introduced new grant resources intended to replace old grant solutions. Here's some of the useful
```

Comment:
I do not have any remarks regarding the content (although I did not run them).

One general remark I have, would be to move this description to separate file - it takes much space here. I would leave the `vX.XX.X -> v0.85.0` with short description and add link to the separate description.